rpc/send.go

```diff
@@ -96,7 +96,9 @@ func (s SendError) CanRetry() bool { return s.canRetry }
 // number of required replies.
 func Send(opts Options, method string, addrs []net.Addr, getArgs func(addr net.Addr) interface{},
 	getReply func() interface{}, context *Context) ([]interface{}, error) {
-	if opts.N < len(addrs) {
+
+	// Return error if provided addrs is less. modified by joezxy 2015/03/07
```

Comment:
The "modified by joezxy" portion of the comment isn't necessary. That's what a revision control system is for (i.e. `git blame`).